braze/src/main/java/com/tealium/remotecommands/braze/BrazeInstance.java

```diff
@@ -603,10 +539,15 @@ public void requestFlush() {
     }
 
     @Override
-    public void registerToken(String token) {
-        if (token == null) return;
+    public void setLastKnownLocation(@NonNull Double latitude, @NonNull Double longitude, @Nullable Double altitude, @Nullable Double accuracy) {
+        if (latitude == null || longitude == null) return;
```

Comment:
Can this if statement be remove with the added `NonNull` annotation in the params? Or is it still needed because params are not enforced/guaranteed to be non null even when annotated?